bloom-housing/bloom
# feat: (re)adding sentry to public site

## Issue Overview


## Description
This readds Sentry to the public site so we can keep track of the errors that are occurring and have been non-replicable 

## How Can This Be Tested/Reviewed?
Make sure you update your .env file to contain the new variable `SENTY_ORG`
Head to http://localhost:3000/sentry-example-page
and press the throw error button (if you have ad-block turned on you may need to turn it off)

head over to sentry and verify the error you threw is showing up there 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
